tests/test_features.py

```diff
@@ -395,3 +396,56 @@ class Truthy(betterproto.Message):
     assert t
     t.bar = 0
     assert not t
+
+
+def test_iso_datetime():
+    @dataclass
+    class Envelop(betterproto.Message):
+        ts: datetime = betterproto.message_field(1)
```

Comment:
Could easily extend this test to cover the repeated case as well, probably like:

```
    tss: List[datetime] = betterproto.message_field(2)
```

Reply:
Added a separate test